build-wasm.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+# Usage: ./build-wasm.sh
+
+set -e
+set -x
+
+# run script from the context of the script-containing directory
+cd "$(dirname $0)"
+
+# This file replicates the instructions found in ../bergamot-translator/README.md under "Build WASM"
```

Comment:
Shouldn't it be `bergamot-translator/README.md` instead of `../bergamot-translator/README.md`?